armi/materials/tests/test_materials.py

```diff
@@ -852,6 +935,13 @@ def test_propertyValidTemperature(self):
 
 
 class Lead_TestCase(_Material_Test, unittest.TestCase):
+    """Unit tests for lead materials.
+
+    .. test:: There is a base class for fluid materials.
+        :id: T_ARMI_MAT_FLUID2
+        :tests: R_ARMI_MAT_FLUID
+    """
```

Comment:
How is lead a fluid? I see that lead.py::Lead is a subclass of Fluid and that's wild to me.

Reply:
Errr I guess this could be for lead cooled reactors....

Reply:
Yeah:

1. Lead-cooled reactors.
2. Lead has a low melting point, and would be liquid in any reactor during operations.